Microsoft.Toolkit.Uwp/Helpers/ConnectionHelper.cs

```diff
@@ -106,5 +109,54 @@ public static ConnectionType ConnectionType
                 return connectionType;
             }
         }
+
+    /// <summary>
+    /// Gets connection ssid for the current Wifi Connection.
+    /// </summary>
+    /// <returns> string value of current ssid/></returns>
+    ///
+    public static async Task<string> GetNetworkSsidAsync()
```

Comment:
This is too specific. We should provide GetNetworkNames method with ConnecitonFilter that allows user to specify whether the connection is WiFi, Data, Ethernet etc.

Please see the implementation I showed in the issue